src/components/Search/Search.jsx

```diff
@@ -11,20 +11,23 @@ type Props = {|
   iconUrl: string,
   instructionText: string,
   onChange: ({target: {value: string}}) => void,
+  onKeyDown: (event: Event) => void,
   onClear: (event: Event) => void,
   placeholder: string,
   value: string,
   data: Data,
 |};
 
 type State = {|
-    text: string
+  text: string
 |};
 
 class Search extends Component<Props, State> {
   static defaultProps = {
     placeholder: 'Search...',
+    onKeyDown: null,
```

Comment:
seems strange that we have to do this

Reply:
this is because React doesn't like `undefined` in props, but asks for `null` instead